From b350c6a823793b00638f88fe25b7a5eca3d49371 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Wed, 2 Dec 2009 18:15:58 -0500
Subject: [PATCH 22/43] Fix error in vbi_get_mem_attr naming

Two different names were used in the original implementation,
vbiMmuAttrGet and vbiMemAttrGet -- the latter was correct,
and the former was an error.  Of course the renaming sed
magic didn't pick up on the typo'd version.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 arch/x86/kernel/vbi/syscalls.S |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/x86/kernel/vbi/syscalls.S b/arch/x86/kernel/vbi/syscalls.S
index 14c3077..ca0a9fd 100644
--- a/arch/x86/kernel/vbi/syscalls.S
+++ b/arch/x86/kernel/vbi/syscalls.S
@@ -500,7 +500,7 @@ vbi_set_mmu_attr:
  * RETURNS: OK or error number in case of failure
  *
  */
-vbiMmuAttrGet:
+vbi_get_mem_attr:
 	movl $VBI_SYS_mmu_attr_get,%eax
 	push $1
 	vmcall
-- 
1.6.5.2

